src/styles/App.scss

```diff
@@ -360,6 +360,7 @@ input[type=number] {
 
     // Thermometer Credit. Adapted from: Copyright (c) 2020 COVID-19 Projections. https://github.com/covid-projections/covid-projections/
 
+    $thermometer-levels: 6;
```

Comment:
Looks like this file slipped in from your other change?

Reply:
Woops. That's weird. I'll fix it.

Reply:
Still here?